README.md

```diff
@@ -1,2 +1,6 @@
-# cattastic
-For all the the cats out there!
+###Instructions
+Pulling from https://thecatapi.com/ we will create a cat comparison web app! 
+
+
+###Learning objective
```

Comment:
Please include a space between the hashtags and the header for proper markdown syntax